### PR TITLE
Plans: replace option cards with daily products

### PR DESCRIPTION
**Pull request by rcanepa**
#### Changes proposed in this Pull Request

* Instead of showing option/generic cards for Jetpack Security and Jetpack Backup, show the daily version of them.
* Remove upsell step from the flow.
* Update Pricing page header.

#### Testing instructions

* Run this PR locally (both envs).
* Visit the Pricing page `http://jetpack.cloud.localhost:3001/pricing`.
* Verify that the header looks like the capture below (we removed the subtitle).
* Verify that Jetpack Security is no longer an option/generic card and instead it is Jetpack Security Daily.
* Verify that Jetpack Backup is no longer an option/generic card and instead it is Jetpack Backup Daily.
* Click on the Jetpack Backup Daily CTA.
* Verify that you were redirected to the checkout page and that you didn't see the upsell page.
* Repeat the steps in Calypso Blue.

Fixes #

##### WordPress.com/plans
image

##### Pricing page
image

### CODE REVIEW

File: client/jetpack-cloud/sections/pricing/header/index.tsx [hunk 1]
```diff
@@ -7,33 +7,41 @@ import { translate } from 'i18n-calypso';
...
-				) }
-				align="center"
-			/>
-			<p>
-				{ translate(
-					'Get everything your site needs, in one package — so you can focus on your business.'
+const Header = () => {
+	const isAlternateSelector = isEnabled( 'plans/alternate-selector' );
+	const header = isAlternateSelector
+		? translate( 'Security, performance, and growth tools for WordPress' )
```
Comment (a8ci18n) [inline client/jetpack-cloud/sections/pricing/header/index.tsx:23]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Complete WordPress security, performance, and growth' )` *ES Score: 6*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/3-col-plans-remove-option-cards)

**Review by mattgawarecki — APPROVED** [review]
Everything looks and behaves as described, and the code looks good as well! 🚢

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/4946923

Thank you @rcanepa for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/4946923).

**Pull request merged**